src/.env.local.js.example

```diff
@@ -39,4 +38,28 @@ module.exports = {
   pageTitle: null,
   // OPTIONAL: The hostname to emulate when testing.
   hostname: "example.local",
+  // OPTIONAL: Machine name of those search fields whose facets/filter and current values should be hidden in UI.
+  // Note: if their values are pre-set (i.e. sent in qs to app), they will still be sent in the query.
+  // hiddenSearchFields: [ // Defaults to [];
+  //   'sm_site_name',
+  //   'ss_federated_type',
+  //   'ds_federated_date',
+  //   'sm_federated_terms',
+  // ],
+  // OPTIONAL: Provides config for adding autocomplete functionality to text search, defaults to false
+  // autocomplete : {
+  //   url: <your-endpoint-for-autocomplete-results>, // required @todo document accepted types
+  //   appendWildcard: true, // OPTIONAL: defaults to false, whether or not to append wildcard to query term
```

Comment:
Says defaults to false but set to TRUE?

Reply:
Thanks, resolved in https://github.com/palantirnet/federated-search-react/pull/26/commits/6d1c966fd4f29ab43059c9efe070bc84207f4f37